### ISSUE TITLE
show accuracy while training classification

### ISSUE THREAD

**Issue by DanTaranis** [PyTorchLightning/lightning-flash#357]
#### What is your question?
How can I see the accuracy per batch along side the loss during training for a simple image classifier.

#### Code

```
# closely following an example from your git page
model = ImageClassifier(backbone="resnet18", num_classes=2)
classifier = flash.Task( model, loss_fn=nn.functional.cross_entropy, optimizer=optim.Adam)
flash.Trainer(gpus=1).fit(classifier, train_set, val_set)
```

#### What have you tried?
I have tried adding metrics=[torchmetrics.Accuracy()] in Task and in model. My understanding is that ImageClassifier is already initialized with an accuracy metric.
 
#### What's your environment?
ubuntu 20.04, conda python 3.7

Thanks !!!

**Comment by ethanwharris** [comment]
Hi @DanTaranis,
Thanks for the question :smiley:

You shouldn't need to wrap the `ImageClassifier` in a `Task` as it is already a `Task`. The following code should work:
```
model = ImageClassifier(backbone="resnet18", num_classes=2, loss_fn=nn.functional.cross_entropy, optimizer=optim.Adam)
flash.Trainer(gpus=1).fit(classifier, train_set, val_set)
```

Can you point me to the example that your code snippet came from?

Hope that helps!

**Comment by DanTaranis** [comment]
Excellent!

I mixed :
"A general task"
and 
"Finetuning"

**Comment by DanTaranis** [comment]
awesome - it worked! accuracy is showing now. 

Thanks so much for the super fast reply.

**Comment by ethanwharris** [comment]
@DanTaranis No worries, glad it's sorted :smiley:
